torch/autograd/grad_mode.py

```diff
@@ -62,6 +62,8 @@ class no_grad(_DecoratorContextManager):
 
     Also functions as a decorator. (Make sure to instantiate with parenthesis.)
 
+    Additionally, consider using :meth:`~torch.nn.eval` when evaluating a model.
```

Comment:
I don't think the link is right here either, it's missing a `Module` no?

Also I would add a bit more details here, what do you think?
Additionally, when working with `nn.Module`, you should consider using `nn.Module.eval` to ensure that `nn.Module` behave properly for evaluation (deactivate dropout, set batchnorm to use saved statistics, etc).